macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -29,7 +29,9 @@ open class AvatarView: NSView {
                       contactImage: NSImage? = nil) {
 
 		// Prefer contactEmail to contactName for uniqueness
-		avatarBackgroundColor = AvatarView.backgroundColor(for: AvatarView.colorIndex(for: contactEmail ?? contactName ?? ""))
+		let color = AvatarView.getColor(for: AvatarView.colorIndex(for: contactEmail ?? contactName ?? ""))
+		avatarBackgroundColor = color.background.resolvedColor()
```

Comment:
pass the window.effectiveAppearance to the resolveColor here too? (and in the next line as well)

Reply:
not passing it will default to window's effective appearance.

Reply:
nevermind this one.. it's in the initializer.

Reply:
updated as per our discussion